src/items/tables/complete_species_table.json

```diff
@@ -0,0 +1,1916 @@
+{
+  "_id": "33udEssZ0Nk7jqtC",
+  "name": "Complete Species table",
+  "description": "This table can be used to create random species tables for locations.\nOr as a base list to create Reincarnation lists from.",
```

Comment:
`This table can be used to generate random species for locations, or to be used as a random species table for the Reincarnation ritual.`

The Reincarnate spell already includes a table, so this should refer to the Ritual instead